src/test/java/com/firebolt/jdbc/client/query/StatementClientImplTest.java

```diff
@@ -45,11 +42,14 @@ class StatementClientImplTest {
 	private FireboltConnection connection;
 
 	@Test
+	@Disabled("Disabled until engine_url is available")
```

Comment:
Which engine url is unavailable here?

Reply:
Everything is available and this test is enabled now.